RAQUASI88/src/keyboard.h

```diff
@@ -173,10 +173,11 @@ enum {
 
   KEY88_INVALID     = 0,
 
-  /* 1〜31 は特殊機能制御用にリサーブしておく */
+  /* 1〜30 は特殊機能制御用にリサーブしておく */
 
   /* 文字キー用の定義 (ASCIIコードに合致) */
 
+  KEY88_DASH        = 31,
```

Comment:
Would it be better to name this KEY88_LONGVOWEL or something similar? Also, shouldn't this be down in the special characters section (i.e. 196)?

Reply:
What is "long vowel" supposed to mean? I'm pretty sure it is a dash or hyphen. I put it closer to where it actually is, i.e. after `KEY88_QUESTION`.

Reply:
https://en.wikipedia.org/wiki/Ch%C5%8Donpu

> ... is a Japanese symbol that indicates a chōon, or a **long vowel** of two morae in length.

Is that not what that key actually is?

Reply:
I searched it and did not find a reference that named the key, so I went for dash. If I find evidence that it is something else I will change it, but it is only an internal name.